waltz-ng/client/measurable/components/summary/measurable-summary.html

```diff
@@ -1,62 +1,62 @@
 <div class="waltz-page-summary waltz-page-summary-attach">
     <div class="row">
-        <div class="col-sm-12 text-muted">
+        <div class="col-sm-2 waltz-display-field-label">
+            Description
+        </div>
+        <div class="col-sm-10 text-muted">
             <waltz-markdown text="$ctrl.measurable.description">
             </waltz-markdown>
-            <br/>
-            <br/>
             <waltz-entity-named-notes-panel parent-entity-ref="$ctrl.entityRef">
             </waltz-entity-named-notes-panel>
-            <hr/>
         </div>
     </div>
 
-    <div class="row">
-        <div class="col-sm-4">
-            <waltz-basic-info-tile name="App Stats"
-
-                                   icon="rocket">
-
-                <div style="font-size: large;"
-                     waltz-jump-to="costs-section">
-                    Cost: <span ng-bind="$ctrl.portfolioCostStr"></span>
-                </div>
-                <div style="font-size: large;"
-                     waltz-jump-to="complexity-section">
-                    Complexity:
-                    <span ng-bind="$ctrl.complexitySummary.cumulativeScore | toFixed:1"></span>
-                </div>
-                <div style="font-size: large;"
-                     ng-if="$ctrl.enrichedServerStats.total > 0"
-                     waltz-jump-to="technologies-section">
-                    Servers #:
-                    <span ng-bind="$ctrl.enrichedServerStats.total"></span>
-                </div>
-            </waltz-basic-info-tile>
-        </div>
 
-        <div class="col-sm-4">
-            <waltz-apps-by-investment-pie applications="$ctrl.applications"
-                                          size="70">
-            </waltz-apps-by-investment-pie>
-        </div>
+    <!-- APP SUMMARY STATS -->
+    <div ng-if="$ctrl.applications.length > 0">
+        <hr>
+        <div class="row" ng-if="$ctrl.applications.length > 0">
+            <div class="col-sm-4">
+                <waltz-basic-info-tile name="App Stats"
```

Comment:
a) why twice (repeated from two lines above)
b) rack and stack the params ;)
c) It looks like the relationship count gets caught up in this.  Should be independently visible based on rel count.